lines/client/rundown.py

```diff
@@ -1,57 +1,89 @@
 """Client for TheRundown API."""
```

Comment:
Might be worth having this in a separate repo.  One repo for a client wrapper, the other a workflow service.